Mods/ModVarietyPack/Defs/ThingDefs/Buildings_Furniture.xml

```diff
@@ -1888,105 +1738,14 @@
-    <researchPrerequisite>PowerI</researchPrerequisite>
-  </ThingDef>
-
-  <ThingDef ParentName="FloorLightStripe">
-    <defName>FloorLightStripe_Red</defName>
-    <label>floor light (stripe) red</label>
-    <graphicData>
-      <texPath>Buildings/Lights/FloorLightStripe_Red</texPath>
-    </graphicData>
-    <description>Stripe floor light integraded into the ground with a red lamp. Waterproof and quite pretty.</description>
-    <comps>
-      <li>
-        <compClass>CompPowerTrader</compClass>
-        <basePowerConsumption>40</basePowerConsumption>
-        <shortCircuitInRain>false</shortCircuitInRain>
-        <soundPowerOn>PowerOnSmall</soundPowerOn>
-        <soundPowerOff>PowerOffSmall</soundPowerOff>
-      </li>
-      <li>
-        <compClass>CompGlower</compClass>
-        <glowRadius>6</glowRadius>
-        <glowColor>(217,0,0,0)</glowColor>
-      </li>
-    </comps>
-    <researchPrerequisite>ColoredLights</researchPrerequisite>
-  </ThingDef>
-
-  <ThingDef ParentName="FloorLightStripe">
-    <defName>FloorLightStripe_Green</defName>
-    <label>floor light (stripe) green</label>
-    <graphicData>
-      <texPath>Buildings/Lights/FloorLightStripe_Green</texPath>
-    </graphicData>
-    <description>Stripe floor light integraded into the ground with a green lamp. Waterproof and quite pretty.</description>
-    <comps>
-      <li>
-        <compClass>CompPowerTrader</compClass>
-        <basePowerConsumption>40</basePowerConsumption>
-        <shortCircuitInRain>false</shortCircuitInRain>
-        <soundPowerOn>PowerOnSmall</soundPowerOn>
-        <soundPowerOff>PowerOffSmall</soundPowerOff>
-      </li>
-      <li>
-        <compClass>CompGlower</compClass>
-        <glowRadius>6</glowRadius>
-        <glowColor>(0,217,0,0)</glowColor>
-      </li>
-    </comps>
-    <researchPrerequisite>ColoredLights</researchPrerequisite>
-  </ThingDef>
-
-  <ThingDef ParentName="FloorLightStripe">
-    <defName>FloorLightStripe_Blue</defName>
-    <label>floor light (stripe) blue</label>
-    <graphicData>
-      <texPath>Buildings/Lights/FloorLightStripe_Blue</texPath>
-    </graphicData>
-    <description>Stripe floor light integraded into the ground with a blue lamp. Waterproof and quite pretty.</description>
-    <comps>
-      <li>
-        <compClass>CompPowerTrader</compClass>
-        <basePowerConsumption>40</basePowerConsumption>
-        <shortCircuitInRain>false</shortCircuitInRain>
-        <soundPowerOn>PowerOnSmall</soundPowerOn>
-        <soundPowerOff>PowerOffSmall</soundPowerOff>
-      </li>
-      <li>
-        <compClass>CompGlower</compClass>
-        <glowRadius>6</glowRadius>
-        <glowColor>(0,0,217,0)</glowColor>
+      <li Class="CommunityCoreLibrary.CompProperties_ColoredLight">
+        <compClass>CommunityCoreLibrary.CompColoredLight</compClass>
       </li>
     </comps>
-    <researchPrerequisite>ColoredLights</researchPrerequisite>
+    <researchPrerequisite>PowerI</researchPrerequisite>
```

Comment:
Yep, the CCL lights are better, much less clutter in the menu, you are right. I will keep it.